rts/Lua/LuaSyncedCtrl.cpp

```diff
@@ -5081,6 +5084,75 @@ int LuaSyncedCtrl::SetFeaturePieceVisible(lua_State* L)
 	return (SetSolidObjectPieceVisible(L, ParseFeature(L, __func__, 1)));
 }
 
+
+/******************************************************************************
+ * Wrecks
+ * @section wrecks
+******************************************************************************/
+
+
+/*** Create a wreck from a unit
+ *
+ * @function Spring.CreateUnitWreck
+ *
+ * @param unitID integer
+ * @param wreckLevel integer Wreck index to use. Default: 1.
+ * @param doSmoke boolean Wreck emits smoke when `true`. Default: `true`
```

Comment:
```suggestion
 * @param doSmoke boolean (Default: `true`) Wreck emits smoke when `true`. 
 ```
 
 Similar for other instances. There's no hard definition for defaults but there's somewhat a convention, let's stick to it, in the future will make it easier to refactor if the spec changes or to implement features that extract the default value from the description.